src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,81 @@
-import React from 'react';
+import React, {
+  ChangeEvent,
+  Dispatch,
+  FormEvent,
+  SetStateAction,
+  useState,
+} from 'react';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
+import cs from 'classnames';
+
+interface Props {
+  movies: Movie[];
+  setMovies: Dispatch<SetStateAction<Movie[]>>;
+}
+
+export const FindMovie: React.FC<Props> = ({ setMovies, movies }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [error, setError] = useState(false);
+  const [isPreview, setIsPreview] = useState(false);
```

Comment:
Are you really need this state?
Can we avoid it? Looks like moviw state enough to make everything what to do isPreview state